scripts/qa_crowbarsetup.sh

```diff
@@ -3404,12 +3404,20 @@ function onadmin_proposal
         done
     fi
     local proposal
-    for proposal in nfs_client pacemaker database rabbitmq keystone swift \
-        ceph monasca glance cinder neutron nova `horizon_barclamp` \
-        ceilometer heat manila trove barbican magnum sahara murano \
-        aodh tempest; do
+    for proposal in nfs_client pacemaker database rabbitmq ; do
         deploy_single_proposal $proposal
     done
+    onadmin_cleanup_db_mq_vips
```

Comment:
Maybe you can comment in the code this split and check? I am not very sure why we do make the clean here